libs/entitlement/bridge/src/main/java/org/elasticsearch/entitlement/bridge/EntitlementChecker.java

```diff
@@ -32,6 +37,15 @@ public interface EntitlementChecker {
 
     void check$java_net_URLClassLoader$(Class<?> callerClass, String name, URL[] urls, ClassLoader parent, URLStreamHandlerFactory factory);
 
+    // "setFactory" methods
+    void check$javax_net_ssl_HttpsURLConnection$setSSLSocketFactory(Class<?> callerClass, HttpsURLConnection conn, SSLSocketFactory sf);
+
+    void check$javax_net_ssl_HttpsURLConnection$$setDefaultSSLSocketFactory(Class<?> callerClass, SSLSocketFactory sf);
+
+    void check$javax_net_ssl_HttpsURLConnection$$setDefaultHostnameVerifier(Class<?> callerClass, HostnameVerifier hv);
+
+    void check$javax_net_ssl_SSLContext$$setDefault(Class<?> callerClass, SSLContext context);
```

Comment:
I'm curious why there are four `check` methods even though this implements just one row of the Entry Points list. Is it because the list contains the method where the actual SM check was performed, and you had to back out to the public caller methods?

Reply:
The other 3 are in "JVM-wide changes"; I know, the classification is arbitrary, but being static these 3 are more problematic (they affects everything), so I thought it was better to make this clear.

Reply:
Oh! I see.